doc/src/site/sphinx/tutorials/sw_automl.rst

```diff
@@ -128,21 +128,21 @@ The following sections describe how to train an AutoML model in Sparkling Water
 
         .. code:: python
 
-            model.transform(testingDF).show()
+            model.transform(testingDF).show(truncate = False)
 
 
 
 Enabling XGBoost Models when Running Sparkling Water on YARN
```

Comment:
Maybe one note: People can run Sparkling Water in different kinds of distributed environments -> Kubernetes, Standalone Spark mode -> the same applies there